packages/psyche/psyche.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "0918nobita"
```

Comment:
it would be great to have an email address as well. Not everyone can or want to contact you through github. Could you add one as well? You can use the form `0918nobita <youremailaddress@mailprovider.org>` for example.

Reply:
I see. I'll add my email address.